ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -249,6 +252,80 @@ Resources:
       Roles:
         - !Ref 'InstanceRole'
 
+  # The EC2 instance writes to it, but this is introduced for the cleanup of the parameter
+  InstanceSSMParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      # This will need to be synced with the `packer-rstudio` project
+      Name: !Sub "/service-catalog/synapse/cred/${LinuxInstance}/odic-accesstoken"
+      Type: String
+      Value: to be replaced by EC2
```

Comment:
I don't think it really matters. It will be replaced with JWT when the user first uses the notebook. The only purpose of this is to ensure cloudformation will delete the SSM Parameter after the notebook is de-provisioned.